universal-login-react/src/ui/Transfer/Transfer.tsx

```diff
@@ -24,27 +24,20 @@ interface ErrorsProps {
   recipientError: boolean;
 }
 
-export const Transfer = ({deployedWallet, transferDetails, updateTransferDetailsWith, tokenDetails, onSendClick, transferClassName}: TransferProps) => {
+export const Transfer = ({deployedWallet, transferDetails, updateTransferDetailsWith, tokenDetailsWithBalance, tokenDetails, onSendClick, transferClassName}: TransferProps) => {
```

Comment:
We can now simplify and simply pass the transferService to Transfer component.
This way we can get rid of `deployedWallet`, `tokenDetails`, `tokenDetailsWithBalance `, `onSendClick` (inline), `transferDetails` (move from props to state), updateTransferDetailsWith (move to current component).